### ISSUE TITLE
-g arg & paired chars as one symbol

### ISSUE THREAD

**Issue by deathmorlock** [tehw0lf/paw#18]
Hello bro, thanks for your job!
Can you add feature, for example
[a] [b] [c] [[d][c0][c)]]
so on 4th position can be "d", "c0", "c)"

**Comment by tehw0lf** [comment]
Heya, thanks for the feedback.

I am not sure whether this would be possible at the moment, as the wordlist generator is using the cartesian product to build all possible combinations per string position, and having two characters in "one position" would break this.

**Comment by deathmorlock** [comment]
Yep... i already saw code and broke my brain with possible variants :D 
Thanks for reply!

**Comment by tehw0lf** [comment]
Sure thing!

Maybe we could work with some kind of sub-grouping there, to say that if we take 'c' for position X, then position X+1 is only allowed to take '0'.

Something like [[c][def]][[0][123]]...which would only allow c to go with 0 , but d, e, f to go with 1, 2, 3.

But I fear that this would impact performance and am not sure if it would actually work.

I will leave the issue open in case anyone finds it and wants to tackle it. As for myself, at the moment I'm rather invested in other projects.
